### DISCUSSION TITLE
The game dont start...

### DISCUSSION THREAD

**Discussion by Jerico3000** [X2CommunityCore/xcom2-launcher#380] [Help]
**AML Version:** 
1.5.0

**Description**
When i click "Run the War of the Chosen" the game just dont start, no error or something, it just dont start... but the firts time i tried to run an errors appears

![Error AML](https://github.com/X2CommunityCore/xcom2-launcher/assets/54676896/a21a0831-83cb-4fd5-a40c-922ad40e8e35)

**Comment by RevZero** [comment]
Do you have some Anti-Virus tool running that may be blocking AML from accessing files?

**Comment by Jerico3000** [comment]
None of sort of, but i did some web surfing looking for another solution and according to another website the problem is that some mods are incompatible with the version of my game...

**Comment by spannerman79** [comment]
If you have anything installed within "Program Files" etc it needs elevated permissions to write, create, delete files and directories.

Move your game/s out of a protected directory, ie `C:\Games\`

**Comment by Jerico3000** [comment]
Thanks, ill try later, after some many attemps i uninstall the game so ill try later...
